pkg/minikube/cluster/cluster_linux.go

```diff
@@ -62,6 +62,26 @@ func createKVMHost(config MachineConfig) *kvmDriver {
 	}
 }
 
+func createKVM2Host(config MachineConfig) *kvmDriver {
+	return &kvmDriver{
```

Comment:
Is there a way that we can reference the driver type directly instead of relying on the extra kvmDriver type that we kind of redefine here?  We'd need to restructure the driver packages a little so that the structs would be in something like `package types` that could be imported without bringing along all the extra libvirt stuff.

Reply:
I think its probably fine for now, but a good TODO